### DISCUSSION TITLE
Additional PayPal Currency Support?

### DISCUSSION THREAD

**Discussion by poppin-fresh** [Summer-16/CSGO-VMPanel#32] [Q&A]
Hello, I'm loving this panel so far. 
Just a quick question regarding the default PayPal currency, would it be possible to add or change the default currency, to say EUR for example? 

I've tried changing the vip_currency to EUR but no luck. 

Thanks!

**Comment by crashzk** [comment]
Wouldn't it be just a change in PayPal? Change to receive in the currency you would like, then PayPal would make the conversion and so on.

**Comment by marcusisdahl** [comment]
@crashzk Setting a standard in PayPal does work fine. I would also like to be able to use another currency NOK as standard and show the prices in NOK on the web panel as well.
